04_UpdateUser.php

```diff
@@ -2,7 +2,7 @@
 <html lang = "eng">
```

Comment:
not much changes in updateuser, just unnecessary spaces and same changes above ^